dag-store.js

```diff
@@ -4,13 +4,10 @@ import uuid from 'node-uuid';
 let nodes = (state = [], action = {}) => {
   switch(action.type) {
```

Comment:
Just curious, is there a usecase that you are having that requires for this? I am not comfortable adding arbitrary properties to the `node` object stored inside the `nodes` array but sounds there seems to be a case!